pandas/io/json/_json.py

```diff
@@ -141,14 +141,25 @@ def to_json(
     default_handler: Callable[[Any], JSONSerializable] | None = None,
     lines: bool = False,
     compression: CompressionOptions = "infer",
-    index: bool = True,
+    index: bool | None = None,
     indent: int = 0,
     storage_options: StorageOptions = None,
     mode: Literal["a", "w"] = "w",
 ) -> str | None:
-    if not index and orient not in ["split", "table"]:
+    if index is None and orient in ["records", "values"]:
```

Comment:
Is this branch necessary? Or can you just change L151 to set it to True for appropriate orients?